src/parser.zig

```diff
@@ -1112,31 +1113,7 @@ pub const Parser = struct {
             },
         }
 
-        // Validate the instruction. Some instructions, e.g. block, loop
-        // are validate separately above.
-        switch (instr) {
-            .block,
-            .loop,
-            .@"if",
-            .br,
-            .br_if,
-            .br_table,
-            .call,
-            .call_indirect,
-            .@"global.get",
-            .@"global.set",
-            .@"local.get",
-            .@"local.set",
-            .@"local.tee",
-            .misc,
-            .@"ref.null",
-            .select_t,
-            .@"table.get",
-            .@"table.set",
-            => {},
-            else => try self.validator.validate(instr),
-        }
-
+        try self.validator.validate(instr);
```

Comment:
Getting rid of this is nice, because this is:
- another place to have remember to add an opcode
- and it is a case of remembering because of the catch all. By removing this the compiler will tell us where we need to update